.nf-core.yml

```diff
@@ -1,8 +1,8 @@
 lint:
   files_exist: false
-    - assets/nf-core-metaval_logo_light.png
-    - docs/images/nf-core-metaval_logo_light.png
-    - docs/images/nf-core-metaval_logo_dark.png
+    - assets/nf-core-meta-val_logo_light.png
```

Comment:
Maybe then nf-core should be dropped from the filename since this is non nf-core?

Reply:
meta-val doesn't contain this file anymore. Here just states that this file doesn't exist to avoid linting issues.

Reply:
Right!